
# [CVE-2022-25304](https://github.com/FreeOpcUa/python-opcua/issues/1466)

## Description

- `All versions of package opcua; all versions of package asyncua are vulnerable to Denial of Service (DoS) due to a missing limitation on the number of received chunks - per single session or in total for all concurrent sessions. An attacker can exploit this vulnerability by sending an unlimited number of huge chunks (e.g. 2GB each) without sending the Final closing chunk.`

## Cvss Data

- **Access Vector**:
  - 
- **Base Score**:
  - 

## Scores

- **Exploitability Score**:
  - 
- **Impact Score**:
  - 
- **Base Severity**:
  - 

## Other Information

- **Publish Date**:
  - 2022-08-23 05:15:07
- **Vulnerability Status**:
  - Analyzed

## References

- **report@snyk.io**: https://github.com/FreeOpcUa/python-opcua/issues/1466
- **report@snyk.io**: https://security.snyk.io/vuln/SNYK-PYTHON-ASYNCUA-2988731
- **report@snyk.io**: https://security.snyk.io/vuln/SNYK-PYTHON-OPCUA-2988730
